runtime/runtime_controller.h

```diff
@@ -660,6 +660,24 @@ class RuntimeController : public PlatformConfigurationClient {
       std::shared_ptr<PlatformIsolateManager>(new PlatformIsolateManager());
   bool has_flushed_runtime_state_ = false;
 
+  // Tracks the views that have been called `Render` during a frame.
+  //
+  // If all registered views (through `AddView`) have been rendered, then the
+  // end of frame will be called immediately, submitting the views to the
+  // pipeline a bit earlier than having to wait for the end of the vsync.
```

Comment:
This is a good comment that is useful context for `RuntimeController::Render`. I would consider moving (or copying) part of this comment to that function.

Reply:
It's hard because this method is actually implementing its parent class `PlatformConfigurationClient::Render`. There are a lot of implementation details that I don't know how to explain in the context of `PlatformConfigurationClient`, or whether we should mention them at all.